l10n-ms/src/main/java/com/example/l10nadmin/controller/L10nController.java

```diff
@@ -33,4 +33,36 @@
 		LOG.info("Retrieve locale texts for {}", locale);
 		return ResponseEntity.ok(l10nService.getLocalization(locale));
 	}
+
+	@GetMapping("l10n")
+	@ApiOperation("Returns all localization entries")
+	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful"),
+			@ApiResponse(code = 401, message = "Unauthorized"),
+			@ApiResponse(code = 500, message = "System Error")})
+	public ResponseEntity<List<L10n>> getAllLocalization() {
+		LOG.info("Retrieve all values");
+		return ResponseEntity.ok(l10nRepository.findAll());
+	}
+
+	@PostMapping("l10n/entry")
+	@ApiOperation("Creates a new entry")
+	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful"),
+			@ApiResponse(code = 401, message = "Unauthorized"),
+			@ApiResponse(code = 500, message = "System Error")})
+	public ResponseEntity postNewEntry(@RequestBody requestForm form) {
```

Comment:
It's a good idea to return the newly created entry after creating it (this way the caller learns its ID)